app/views/activity_forms/is_oda.html.haml

```diff
@@ -0,0 +1,6 @@
+= render layout: "wrapper" do |f|
+  = f.govuk_error_summary
+  = f.govuk_radio_buttons_fieldset(:is_oda, legend: { text: t("form.legend.activity.is_oda"), size: "xl", tag: "h1" }) do
+    = f.hidden_field :is_oda, value: nil
```

Comment:
I'm not sure I understand - what does this hidden field do?

Reply:
It makes sure that the `params` are submitted even if the user presses Continue without selecting a radio button. Otherwise we get no payload, and then `param is missing or the value is empty: activity`.

Reply:
I've added some detail about this in the commit message.